phpBB/styles/prosilver/template/bbcode.html

```diff
@@ -11,6 +11,39 @@
 <!-- BEGIN quote_username_open --><blockquote><div><cite>{USERNAME} {L_WROTE}{L_COLON}</cite><!-- END quote_username_open -->
 <!-- BEGIN quote_open --><blockquote class="uncited"><div><!-- END quote_open -->
 <!-- BEGIN quote_close --></div></blockquote><!-- END quote_close -->
+<!-- BEGIN quote_extended -->
+<blockquote>
+	<xsl:if test="not(@author)">
```

Comment:
You should accept if the user only fills some of the parameters. Otherwise it makes no real sense in the concept. You shouldn't reject a bbcode tag's information if a parameter is missing if they all are optional and not inherently depended on oneanother.